antarest/study/storage/variantstudy/model/model.py

```diff
@@ -1,22 +1,49 @@
 import typing as t
+import uuid
 
+import typing_extensions as te
 from pydantic import BaseModel
 
 from antarest.core.model import JSON
 from antarest.study.model import StudyMetadataDTO
 
+LegacyDetailsDTO = t.Tuple[str, bool, str]
+"""
+Legacy details DTO: triplet of name, output status and output message.
+"""
+
+
+class NewDetailsDTO(te.TypedDict):
+    """
+    New details DTO: dictionary with keys 'id', 'name', 'status' and 'msg'.
+
+    Attributes:
+        id: identifiant de la commande (UUID),
+        name: nom de la commande,
+        status: statut de la commande (true ou false),
+        msg: message de la génération de la commande ou message d'erreur (si le statut est false).
+    """
+
+    id: t.Optional[uuid.UUID]
```

Comment:
An optional id seems a bit weird, it's for backward compatibility ? if so why not adding automatic UUID generation for legacy commands or run a migration ?

Reply:
otherwise the new format looks better 👍🏼